dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -33,6 +33,7 @@ float AnimatedVisualPlayer::AnimationPlay::FromProgress()
     return m_fromProgress;
 }
 
+// REENTRANCE SIDE EFFECT: IsPlaying DP.
```

Comment:
These reentrancy tracking issues seems like it might easier to nail down if this control included Tracing.  You can see Scroller, SwipeControl, or PullToRefresh as examples of controls which already have tracing built it.

Reply:
If you wanted/needed additional tools, I'm not saying you need to do that for this check in.

---
In reply to: [343379771](https://github.com/microsoft/microsoft-ui-xaml/pull/1558#discussion_r343379771) [](ancestors = 343379771)

Reply:
Can you expand on this... are you saying to add tracing into the code, and then what do we do with the result?

---
In reply to: [343381461](https://github.com/microsoft/microsoft-ui-xaml/pull/1558#discussion_r343381461) [](ancestors = 343381461,343379771)

Reply:
if you replaced comments in the code with tracing statements, when you hit a dependency reentrency crash the tracing output will tell you which potential causes you hit for that repro, instead of having to look at every call in the stack trace.

Reply:
I see. That could be useful but we still wouldn't know that the behavior the caller is seeing isn't what they expect. 

---
In reply to: [343775739](https://github.com/microsoft/microsoft-ui-xaml/pull/1558#discussion_r343775739) [](ancestors = 343775739)